esx_service/vmci/vmci_client.c

```diff
@@ -161,9 +165,10 @@ dummy_get_reply(be_sock_id *id, be_request *r, be_answer* a)
 static be_sock_status
 vsock_init(be_sock_id *id, int cid, int port)
 {
+   static int round_robin = START_CLIENT_PORT;
```

Comment:
nit: maybe just name it client_port and add "// selected using round-robin" to the coments?